maximize-select2-height.js

```diff
@@ -67,16 +51,45 @@
   //   a page)
   // @return {Number} the maximum height of the Select2 results box to display
   var computeMaxHeight = function (
-    $select2Results, options, dropdownDownwards
+    id, $select2Results, $grandparent, options, dropdownDownwards
   ) {
     var height;
-    var pixelsAboveDropdown = topOffset($select2Results) - $window.scrollTop();
+    var resultsBoxMiscellaniaHeight;
+    var widgetBoxOffset;
 
     if (dropdownDownwards) {
+      // When the dropdown appears downwards, the formula is:
+      //   total window size
+      // - size of content above results box
+      // - out-of-window pixels we've scrolled past
+      // ------------------------------------------
+      //   total height available to us
+
       // innerHeight is more accurate across browsers than $(window).height().
-      height = window.innerHeight - pixelsAboveDropdown;
+      height = window.innerHeight -
+               $select2Results.offset().top -
```

Comment:
Same. Trailing `-` should be `+`.
Or add parens before `$select2Results.offset().top - ...`